Sources/MenuContainerViewController.swift

```diff
@@ -18,18 +18,31 @@
 
 import UIKit
 
+/**
+ Container for menu view controller.
+ */
 open class MenuContainerViewController: UIViewController {
 
+    /**
+     The view controller for side menu.
+     */
     public var menuViewController: MenuViewController! {
         didSet {
             if menuViewController == nil {
-                fatalError("Invalid 'menuViewController' value. It should not be nil")
+                fatalError("Invalid menuViewController value. It should not be nil")
             }
             menuViewController.menuContainerViewController = self
             menuViewController.transitioningDelegate = self.navigationMenuTransitionDelegate
             menuViewController.navigationMenuTransitionDelegate = self.navigationMenuTransitionDelegate
         }
     }
+    
+    /**
+     The options defining side menu transitioning.
+     
+     Could be set at any time of controller lifecycle.
+     Are passed to MenuInteractiveTransition class.
```

Comment:
> Are passed to MenuInteractiveTransition class

I think it describes implementation details and we should exclude it If `MenuInteractiveTransition` is not public API.